guia/extensões.md

```diff
@@ -1 +1,19 @@
 ## Extensões
+
+Extensões adicionam novas funcionalidades para uma classe existente, estrutura, enumeração ou tipo de protocolo. Isso inclui a capacidade de estender tipos pelos quais você não possui acesso ao código fonte original (conhecido como modelo retroativo). 
```

Comment:
`Extensões` aqui deve ser entre underline (`_`) pra respeitar o itálico do texto original.